src/test/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheckTest.java

```diff
@@ -827,4 +827,150 @@ public void testSwitchExpression7() throws Exception {
         verifyWithInlineConfigParser(
                 getNonCompilablePath("InputRightCurlyTestSwitchExpression7.java"), expected);
     }
+
+    @Test
+    public void testCaseBlocksInSwitchStatement1() throws Exception {
```

Comment:
Please name test inputs/methods in some way to make it obvious what our motivation is for each test. Also, `Test` in the file name is redundant and makes these file names longer than they need to be. The next developer to come to this file should be able to easily look at the test method name and know what it's intention is. We can draw inspiration from the code under test, the configuration, some bug, etc. For instance, this method could be called `testCaseBlocksInSwitchStatementAlone` to give us an obvious hint as to what the configuration is.

Reply:
@nrmancuso done